src/tests/ftest/io/seg_count.py

```diff
@@ -100,7 +100,7 @@ def test_segcount(self):
         Use Cases: Different combinations of 32/64/128 Clients, 8b/1k/4k
                    record size, 1k/4k/1m/8m transfersize and stripesize
                    and 16 async io.
-        :avocado: tags=ior,eightservers,ior_segcount,performance
+        :avocado: tags=all,mpiio,large,ior_segcount
```

Comment:
This test is actually > large as it wants 8 servers and 2 clients, making it > 8 nodes which is what large is currently.

Reply:
Still wondering about this one.

Reply:
As per Scotts excel sheet presentation once in our QA meeting regarding tags, the description of large is 12 nodes and medium is 9 nodes.